server/runtime/src/main/server/docs/examples/README.adoc

```diff
@@ -0,0 +1,15 @@
+[id='server_configuration_examples']
```

Comment:
should be a text file IMO

Reply:
I'll leave that to @oraNod

Reply:
fair enough @pruivo I'll make some changes.